src/accelerate/data_loader.py

```diff
@@ -421,6 +524,13 @@ def prepare_data_loader(
     if new_batch_sampler is None:
         kwargs["batch_size"] = dataloader.batch_size // num_processes if split_batches else dataloader.batch_size
 
+    if central_dataloader:
+        data_loader = DataLoaderDispatcher(
+            new_dataset, split_batches=split_batches, batch_sampler=new_batch_sampler, **kwargs
+        )
+        data_loader.my_batch_size = dataloader.batch_size
```

Comment:
What is this `my_batch_size` used for?

Reply:
Woopsie, this is some leftover to clean up! Thanks for flagging!